const.go

```diff
@@ -28,17 +28,23 @@ const MessageTypeDetachedSignature MessageType = 2
 // signcrypted message.
 const MessageTypeSigncryption MessageType = 3
 
+// Version1 returns the Version for Saltpack V1.
 func Version1() Version {
 	return Version{Major: 1, Minor: 0}
 }
+
+// Version2 returns the Version for Saltpack V2.
 func Version2() Version {
 	return Version{Major: 2, Minor: 0}
 }
 
+// CurrentVersion returns the Version for the currently-used Saltpack
+// version.
 func CurrentVersion() Version {
-	return Version1()
+	return Version2()
```

Comment:
Time for this, right?

Reply:
It looks like we always explicitly supply the version in `client`, so this will only affect hypothetical 3rd party callers who keep the default? It does seem like we should bump it, if that's right.

Reply:
Damn shame that Go doesn't give us a way to signify this kind of change in the library version...

Reply:
Yes, that's correct. And yeah, what a shame...